Makefile

```diff
@@ -7,7 +7,7 @@ guard-%:
         exit 1; \
     fi
 generate: guard-GOPATH
-	git submodule update --init --recursive
+	git submodule update --init --recursive --remote
```

Comment:
> Instead of using the superproject’s recorded SHA-1 to update the submodule, use the status of the submodule’s remote-tracking branch. 

This is a very dangerous change to add and breaks immutability of the build process @toddbaert

Reply:
Yes, this makes sense. I didn't think about the nuance of this change. Thank you, I will revert this.